tests/components/Pagination.test.tsx

```diff
@@ -0,0 +1,168 @@
+import { render, screen, within } from '@testing-library/react';
+import { State, VerticalSearchState } from '@yext/answers-headless-react';
+import { spyOnActions, mockAnswersState, mockAnswersHooks } from '../__utils__/mocks';
+import userEvent from '@testing-library/user-event';
+import { Pagination } from '../../src/components/Pagination';
+import { verticalNoResults } from '../__fixtures__/data/noresults';
```

Comment:
nit: maybe we can make a vertical/ folder in data/ and have noresults and results file? having just noresults may be a bit less clear that this related to vertical results only, not universal as well

Reply:
sure, done!